creme/geolocation/utils.py

```diff
@@ -113,14 +113,23 @@ def addresses_from_persons(queryset, user):
 
 
 def get_radius():
-    return SettingValue.objects.get_4_key(
+    return SettingValue.objects.value_4_key(
         setting_keys.NEIGHBOURHOOD_DISTANCE,
-        default=DEFAULT_SEPARATING_NEIGHBOURS,
-    ).value
+        default=constants.DEFAULT_SEPARATING_NEIGHBOURS,
+    )
 
 
 def get_google_api_key():
-    return SettingValue.objects.get_4_key(setting_keys.GOOGLE_API_KEY, default='').value or ''
+    return SettingValue.objects.value_4_key(setting_keys.GOOGLE_API_KEY, default='') or ''
```

Comment:
`or ''` useless now, isn't it ?

Reply:
Not useless as empty value is allowed and "cast" as None by SettingValue (there is a TODO about that issue in the code)

Reply:
OK ; a comment/TODO would be nice.